File: code_snippets/java/src/main/java/develop/ServingHttp.java
package develop;

// <start_here>
import dev.restate.sdk.endpoint.Endpoint;
import dev.restate.sdk.http.vertx.RestateHttpServer;

class MyApp {
  public static void main(String[] args) {
    RestateHttpServer.listen(
        Endpoint.builder()
            .bind(new MyService())
            .bind(new MyVirtualObject())
            .bind(new MyWorkflow())
            .build());
  }
}
// <end_here>

File: code_snippets/kotlin/src/main/kotlin/develop/ServingHttp.kt
package develop

import dev.restate.sdk.http.vertx.RestateHttpServer
import dev.restate.sdk.kotlin.endpoint.endpoint

// <start_here>

fun main() {
  RestateHttpServer.listen(
      endpoint {
        bind(MyService())
        bind(MyVirtualObject())
        bind(MyWorkflow())
      })
}
// <end_here>
